utils/notify/publish_notification.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+from slack_sdk import WebClient
+from slack_sdk.models import blocks
+
+
+class Settings:
+    channel: str
+    workspace: str
```

Comment:
Does this get used?

Reply:
it's used a bit further down to set the values for each when instantiating the notifier object; I've kept it just to stay aligned with `ci-image-builder` in case we bring it into `platform-tools`

otherwise no, not strictly needed

Reply:
Perhaps we could remove for now? Up to you. Not a blocker

Reply:
@gabelton sorted, fancy giving it a quick re-review?